File: 3Week/14Day/java/Algo06/src/test2/nineteen.java
package test2;

public class nineteen {
	public String solution(String my_string, int n) {
        StringBuilder answer = new StringBuilder();
        
        for(String str : my_string.split("")) {
        	for(int i = 0; i < n; i++) {
        		answer.append(str);
        	}
        }
        return answer.toString();
    }
}

File: 3Week/15Day/과제2/nineteen.py
def solution(price):
    if price >= 500000: # 50만원
        return int(price * 0.80)
    elif price >= 300000: # 30만원
        return int(price * 0.90)
    elif price >= 100000: # 10만원
        return int(price * 0.95)
    return price